icu4c/source/i18n/unicode/unumberformatter.h

```diff
@@ -268,8 +268,6 @@ typedef enum UNumberGroupingStrategy {
 #endif  /* U_HIDE_INTERNAL_API */
 
 } UNumberGroupingStrategy;
-
-
 #endif  /* U_HIDE_DRAFT_API */
 
 #ifndef U_HIDE_DRAFT_API
```

Comment:
I think everything in this file can be promoted.

Reply:
…except for the two @draft ICU 64s